scenarios/10_Siege_of_Haeltin.cfg

```diff
@@ -258,9 +277,72 @@
             [/have_unit]
             [then]
                 {MSG_Reumario _"We really should sacrifice an ox to God. There are so many of them; without God's help we won't be able to defeat them."}
+                {MSGW_Yahyazad _"We don't even have any oxen, though."}
+                {MSG_Reumario _"Well, something similar, then."}
+                [unit]
+                    type=Woodland Boar
+                    side=5
+                    id=sacrificial_boar
+                    x,y=4,4
+                    animate=yes
+                    [ai]
+                        [micro_ai]
+                            ai_type=forest_animals
+                            tusker_type=Woodland Boar
+                            [filter_location]
+                                terrain=*^F*,Gg,Hh,Re,Ss
+                            [/filter_location]
+                        [/micro_ai]
+                    [/ai]
+                [/unit]
+                {MSG_Reumario _"...like that boar! Let me go capture it, sacrifice it, and then surely God's favor will be upon us!"}
+                [event]
+                    name=last breath
+                    [filter]
+                        id=sacrificial_boar
+                    [/filter]
+                    [filter_second]
+                        id=Reumario
+                    [/filter_second]
+                    {MSG_Reumario _"I have caught it! Please, O God, accept this boar as a sacrifice, and look with blessing upon us, your children!"}
```

Comment:
```suggestion
                    {MSG_Reumario _"Please, O God, accept this boar and look upon your children with your blessings!"}
```

Reply:
Changes to the second sentence are fine, but the removal of the first sentence makes the transition too harsh; it feels like jumping straight into the middle of a conversation, rather than starting fresh from just having done something

Reply:
Okay. You can just change the second sentence.

Reply:
ok, so, uh, let's see, how do I do this... I guess if I edit your suggestion...?